src/System.Private.CoreLib/shared/Interop/Windows/BCrypt/Interop.BCryptGenRandom.cs

```diff
@@ -19,19 +19,17 @@ internal static unsafe int BCryptGenRandom(byte* pbBuffer, int count)
         }
 
         private const int BCRYPT_USE_SYSTEM_PREFERRED_RNG = 0x00000002;
-        internal const int STATUS_SUCCESS = 0x0;
-        internal const int STATUS_NO_MEMORY = unchecked((int)0xC0000017);
 
         [DllImport(Libraries.BCrypt, CharSet = CharSet.Unicode)]
-        private static extern unsafe int BCryptGenRandom(IntPtr hAlgorithm, byte* pbBuffer, int cbBuffer, int dwFlags);
+        private static extern unsafe NTSTATUS BCryptGenRandom(IntPtr hAlgorithm, byte* pbBuffer, int cbBuffer, int dwFlags);
     }
 
     internal static unsafe void GetRandomBytes(byte* buffer, int length)
```

Comment:
This part is used for CoreLib only. Move it to a separate file, like `Interop.BCryptGenRandom.GetRandomBytes.cs` ?